movies/src/__tests__/app.spec.js

```diff
@@ -0,0 +1,36 @@
+const request = require('supertest');
+const app = require('../app');
+
+describe('app', function () {
+
+  describe('entrypoint', function () {
```

Comment:
Can we have function () replaced with arrow functions?

Reply:
Yes sorry, need to update my live template